### PR TITLE
Say an error message when the reminder parsing failed. r=samgiles

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by julienw** [comment]
Put "Closes #53" in the commit log :)

File: app/js/views/reminders.jsx [hunk 1]
```diff
@@ -104,6 +107,10 @@ export default class Reminders extends React.Component {
       });
   }
 
+  onParsingFailure() {
+    this.speechController.speak(`I didn't understand that. Can you repeat?`);
```
Comment (samgiles) [inline app/js/views/reminders.jsx:114]: Should we immediately start the speech recognition again?  And maybe understand something like "never mind" or "it doesn't matter" to cancel?

Comment (gmarty) [inline app/js/views/reminders.jsx:114]: Ah very good point!
We'll definitely need a ping/pong question and answers mechanism with preserved context for the lists. We can also use that when an information is missing from a reminder (Q: "Remind me of..." A: "When do you want me to remind you?"). When such a logic is in place we can totally improve this parsing failing scenario and have Cue to listen for a second phrase immediately.
I'd like to delay implementing this for now and take the time to build something solid.
How does it sound?

Comment (samgiles) [inline app/js/views/reminders.jsx:114]: Yes, sounds good to me.  We should make an issue for it.

Comment (gmarty) [inline app/js/views/reminders.jsx:114]: I'm not convinced we need an issue for that. That'll be one item in the task list of the lists feature implementation. Let's wait until the scope of this new feature is more defined.

**Pull request merged**